actionpack/lib/action_controller/metal/etag_with_template_digest.rb

```diff
@@ -25,7 +25,7 @@ module EtagWithTemplateDigest
       class_attribute :etag_with_template_digest
       self.etag_with_template_digest = true
 
-      ActiveSupport.on_load :action_view, yield: true do
+      ActiveSupport.on_load :action_controller, yield: true do
```

Comment:
@rafaelfranca It sounds like you have an exact idea of how you want to do this.  I want to avoid iterating several times, each time getting a little closer, when you already know what you want to see.

Can you please describe to me exactly what you have in mind, if you don't have time to just do this yourself?

For instance, I'm not sure whether this should be in an initializer, or after the Railtie class is closed.

I'm also not sure whether at that point, we need to add it to an included block, or just iterate over `ActionController::Base.descendants` and extend each one.